src/controllers/users/handlers/get.productsLocation.handler.ts

```diff
@@ -0,0 +1,13 @@
+import { RequestContext, wrap } from "@mikro-orm/core";
+import { ProductGiftBody } from "../../../contracts/product.body";
+import { Product } from "../../../entities/product.entity";
+import { User } from "../../../entities/user.entity";
+import { Fridge } from "../../../entities/fridge.entity";
+
+export const getProductsLocation = async (userId: string, location: string) => {
+    const em = RequestContext.getEntityManager();
+    const fridgeIds = []
+    const fridges = await em.find(Fridge, { location: Number(location) });
+    fridges.map(fridge => fridgeIds.push(fridge.id));
```

Comment:
Not the correct usage:
either:
```ts
const fridgeIds = []

fridges.forEach((fridge) => {
  fridgeIds.push(fridge.id);
});
```

or 

```ts
const fridgeIds = fridges.map((fridge) => fridge.id)

```

[Array.map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)
[Array.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)

![0_yUVO_gUHBjxYuAl8](https://github.com/BsA-Bootin/fridge-panenco/assets/72884674/5dc697df-2dcb-4f27-b933-8bc4bd1fcbde)